ui-tests/cypress/support/pageObjects/administration/languages_Labels/Languages_LabelsPage.js

```diff
@@ -98,4 +109,88 @@ export default class Languages_LabelsPage extends Content {
   getDisplayedLabelsCount() {
     return this.getDisplayedLabelsTable().find(this.displayedLabelRow)
   }
+
+  getActiveLanguageSelector() {
+    return this.getContents().find(this.languageNavigation)
+  }
+
+  getLabelPaginationForm() {
+    return this.getContents().find(this.labelPaginationForm)
+  }
+
+  getLabelPaginationTextArea() {
+    return this.getLabelPaginationForm().find(this.labelPaginationInput)
+  }
+
+  getLabelRowByCode(code) {
```

Comment:
This should return the label row, as per method name, not a button inside it...

Reply:
Done